### PR TITLE
agent FAQ migration

### PR DESCRIPTION
**Pull request by apigirl**
### What does this PR do?

Refreshes the Agent FAQs.

### Motivation

Better docs for a better world!

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by apigirl** [comment]
To do still:
- [ ] fix broken links
- [ ] update FAQ page
- [ ] update guide pages
- [ ] move images

**Comment by apigirl** [comment]
After this gets merged, I need to update the uninstall hotjar

File: content/en/agent/guide/agent-5-kubernetes-basic-agent-usage.md [hunk 1]
```diff
@@ -299,5 +299,5 @@ Checks
 [13]: https://quay.io/coreos/kube-state-metrics
 [14]: https://github.com/DataDog/integrations-core/blob/master/kubernetes_state/datadog_checks/kubernetes_state/data/conf.yaml.example
 [15]: https://yum.datadoghq.com/stable/6/x86_64
-[16]: /agent/faq/how-do-i-install-the-agent-on-a-server-with-limited-internet-connectivity/
+[16]: /agent/guide/how-do-i-install-the-agent-on-a-server-with-limited-internet-connectivity/
```
Comment (maycmlee) [inline content/en/agent/guide/agent-5-kubernetes-basic-agent-usage.md:302]: A broken link.

File: content/fr/agent/guide/agent-5-kubernetes-basic-agent-usage.md [hunk 2]
```diff
@@ -298,5 +298,5 @@ Checks
 [13]: https://quay.io/coreos/kube-state-metrics
 [14]: https://github.com/DataDog/integrations-core/blob/master/kubernetes_state/datadog_checks/kubernetes_state/data/conf.yaml.example
 [15]: https://yum.datadoghq.com/stable/6/x86_64
-[16]: /fr/agent/faq/how-do-i-install-the-agent-on-a-server-with-limited-internet-connectivity/
+[16]: /fr/agent/guide/how-do-i-install-the-agent-on-a-server-with-limited-internet-connectivity/
```
Comment (maycmlee) [inline content/fr/agent/guide/agent-5-kubernetes-basic-agent-usage.md:301]: Same as above, broken link.

File: content/ja/agent/guide/agent-5-kubernetes-basic-agent-usage.md [hunk 3]
```diff
@@ -298,5 +298,5 @@ Checks
 [13]: https://quay.io/coreos/kube-state-metrics
 [14]: https://github.com/DataDog/integrations-core/blob/master/kubernetes_state/datadog_checks/kubernetes_state/data/conf.yaml.example
 [15]: https://yum.datadoghq.com/stable/6/x86_64
-[16]: /ja/agent/faq/how-do-i-install-the-agent-on-a-server-with-limited-internet-connectivity/
+[16]: /ja/agent/guide/how-do-i-install-the-agent-on-a-server-with-limited-internet-connectivity/
```
Comment (maycmlee) [inline content/ja/agent/guide/agent-5-kubernetes-basic-agent-usage.md:301]: Same as above, broken link.

**Review by maycmlee — COMMENTED** [review]
Hey @apigirl! Just a couple of broken links! Otherwise looks good!

**Review by maycmlee — APPROVED** [review]

**Pull request merged**